src/Components/profiles/MissionsList.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import MissionItem from './MissionItem';
+
+const MissionsList = (props) => {
+  const { lists, mission, count } = props;
+  return (
+    <div className="w-50 me-3">
+      <h3 className="h3">My Missions</h3>
+      <ul className="list-group">
+        {
+          lists.map((next) => (
+            <MissionItem
+              key={next.mission_id}
+              names={next.mission_name}
+              ids={next.mission_id}
+              missions={mission}
+              wikis={next.wiki}
+            />
+          ))
+        }
+      </ul>
+      {
+      (count === 0) && <p className="p-3 lead">No Missions Reserved Yet</p>
+    }
+    </div>
```

Comment:
- [ ] Nice implementation of this placeholder text when there are no reserved missions 💯